google/cloud/forseti/services/scanner/dao.py

```diff
@@ -190,3 +201,40 @@ def map_by_resource(violation_rows):
             v_by_type[v_resource].append(v_data)
 
     return dict(v_by_type)
+
+def _create_violation_hash(violation_full_name, inventory_data, violation_data):
+    """Create a hash of violation data using the requested algorithm.
```

Comment:
Should remove the comment about the requested algorithm.

Reply:
done